.env.sample

```diff
@@ -2,5 +2,7 @@ NODE_ENV=development
 MONGO_URI=mongodb://localhost:27017/codinggardencommunity
 TEST_MONGO_URI=mongodb://localhost:27017/codinggardencommunity-test
 ADMIN_SECRET=keyboardcat
+YOUTUBE_API_KEY=aaaaaaaaaaaaaaaaaaaaaaaaaaa
+YOUTUBE_CHANNEL_ID=UCLNgu_OupwoeESgtab33CCw
```

Comment:
The youtube channel ID can be hard coded, it doesn't have to be an environment variable.